hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionMetrics.java

```diff
@@ -33,6 +37,30 @@ public void setInflight(long inflight) {
         this.inflight = inflight;
     }
 
+    public String getTimeoutsM1() {
+        return timeoutsM1;
+    }
+
+    public void setTimeoutsM1(String timeoutsM1) {
```

Comment:
sicne you are tinkering with this class, how about some boyscout approach and making it immutable?